This is already part of liboxygenstyle

--- origsrc/oxygen-5.8.3/kdecoration/CMakeLists.txt	2016-11-01 07:17:25.000000000 -0500
+++ src/oxygen-5.8.3/kdecoration/CMakeLists.txt	2016-11-09 01:35:49.387005500 -0600
@@ -53,7 +53,6 @@ set(oxygendecoration_config_SRCS
     config/oxygenexceptiondialog.cpp
     config/oxygenexceptionlistwidget.cpp
     config/oxygenexceptionmodel.cpp
-    config/oxygenitemmodel.cpp
 )
 
 set(oxygendecoration_config_PART_FORMS
